gulpfile.js

```diff
@@ -460,6 +528,8 @@ gulp.task('release', ['apps', 'clean-release'], function () {
     if (platforms.indexOf('win64') !== -1) {
         releaseWin('win64');
     }
+    
+    return merged.isEmpty() ? null : merged;
```

Comment:
Probably also need to replace `null` for `done()` here, but I'm not sure about that (need to test).

Reply:
As I said in the other similar comment, it will be more correct.